dll/cpl/desk/lang/pt-PT.rc

```diff
@@ -36,26 +36,26 @@ BEGIN
     PUSHBUTTON "&Personalizar...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
 END
 
-IDD_SCREENSAVER DIALOGEX 0, 0, 246, 216
+IDD_SCREENSAVER DIALOGEX 0, 0, 246, 202
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Protecção de ecrã"
 FONT 8, "MS Shell Dlg", 0, 0
 BEGIN
     CONTROL "", IDC_SCREENS_PREVIEW, "Static", SS_OWNERDRAW, 70, 10, 105, 74
-    GROUPBOX "Pro&tecção de ecrã", IDC_SCREENS_DUMMY, 8, 92, 230, 64
+    GROUPBOX "Pro&tecção de ecrã", IDC_SCREENS_DUMMY, 8, 92, 230, 52
     COMBOBOX IDC_SCREENS_LIST, 14, 103, 100, 100, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_TABSTOP
     PUSHBUTTON "&Definições", IDC_SCREENS_SETTINGS, 120, 102, 50, 15, WS_GROUP
     PUSHBUTTON "&Pré-visualizar", IDC_SCREENS_TESTSC, 175, 102, 50, 15
-    LTEXT "Ag&uardar:", IDC_WAITTEXT, 14, 125, 30, 9
-    EDITTEXT IDC_SCREENS_TIMEDELAY, 46, 123, 32, 13, ES_RIGHT | WS_GROUP
+    LTEXT "Ag&uardar:", IDC_WAITTEXT, 14, 125, 32, 9
+    EDITTEXT IDC_SCREENS_TIMEDELAY, 48, 123, 32, 13, ES_RIGHT | WS_GROUP
     CONTROL "", IDC_SCREENS_TIME, UPDOWN_CLASS, UDS_SETBUDDYINT | UDS_ALIGNRIGHT |
             UDS_AUTOBUDDY | UDS_ARROWKEYS | WS_BORDER | WS_GROUP, 56, 123, 12, 13
-    LTEXT "minutos", IDC_MINTEXT, 80, 125, 26, 9
-    CONTROL "Ao &reiniciar, proteger por palavra-passe", IDC_SCREENS_USEPASSCHK, "button",
-            BS_AUTOCHECKBOX | WS_TABSTOP, 30, 140, 200, 11
-    GROUPBOX "Energia do monitor", IDC_SCREENS_DUMMY2, 8, 164, 230, 41
-    LTEXT "Para ajustar as definições de energia do monitor e poupar energia, clique em 'Energia'.", IDC_STATIC, 16, 174, 146, 27
-    PUSHBUTTON "&Energia...", IDC_SCREENS_POWER_BUTTON, 167, 179, 61, 15
+    LTEXT "minutos", IDC_MINTEXT, 85, 125, 26, 9
+    CONTROL "Pedi&r senha no currículo", IDC_SCREENS_USEPASSCHK, "button",
```

Comment:
What happened with the translation change? @zecarlos1957 ?

Reply:
It didn't fit the dlgs dimension.

Reply:
Where did you see this horrible text?
the current text on this  CONTROL is "Ao &reiniciar, proteger por palavra-passe"

Reply:
lol then increase the size instead of inventing translations ^^

Reply:
Why do you want to modify this text?
![cpl](https://github.com/user-attachments/assets/ec3d2203-4a7b-432b-a979-db51d2422e46)

Reply:
Because you changed the whole dialogs dimensions to add it there. After my change it looks much better and avoids changing the dialogs dimensions (they are back in sync with en-US afterwards), see https://github.com/reactos/reactos/pull/7298#issuecomment-2326794547 for the after2-pic.

Reply:
Changing only the checkbox control size isn't enough ?

Reply:
no. Look at the after2 pic and compare it with zecarlos pic. zecarlos changed all the positions in the dlg for his new layout, including the dlgs height. That is not good. especially because this dialog is part of a property-sheet. He can not simply enlarge one of its dialogs to a bigger height.

Reply:
Ah, so we would have to revert to the original size + position and then do the multiline thing. or whatever

Reply:
what I do currently use seems to make a bit of sense to me:
![image](https://github.com/user-attachments/assets/4e05c526-47ab-4185-b768-32766046435c)

But what you propose @zecarlos1957 "Ao &reiniciar, proteger por palavra-passe" does not make sense to me:
![image](https://github.com/user-attachments/assets/bc2fd552-4100-48a9-b9ac-6b4e4a668780)

That is very misleading. Because that has nothing to do with rebooting or restart actually.
Or would you translate that any different than the computer does?

For the record: The french text matches the available space fine. It doesn't need to be touched at all.
![image](https://github.com/user-attachments/assets/2721396d-4ccb-4625-8532-06398ac97226)

Reply:
None Portuguese understand the content of this message. This has no sense in Portuguese. It's probably an encrypted message that only Joaquim understands.
The correct translation for the Portuguese is "Ao iniciar, proteger por palavra-passe" as it says in windows 2003 and XP.

![cpl](https://github.com/user-attachments/assets/57372c83-fe3c-48fd-a215-318c08f7b01d)

Reply:
Can you provide a screenshot of the pt-PT dlg **from MS**?

Reply:
OK. Thanks for your attention.

![cpl_2003](https://github.com/user-attachments/assets/8ac3cbef-82c7-450a-829a-60ad38fc72cc)

Reply:
Interesting to note is that MS increased the horizontal size of that dlg for pt-PT. Not very desireable. I came up with the following for master head:
![image](https://github.com/user-attachments/assets/b8bc4660-07a2-4778-838d-59175e02f140)
But I will not apply that solution for older branches.
Resolved.